airbyte-cdk/java/airbyte-cdk/db-destinations/src/main/kotlin/io/airbyte/cdk/integrations/destination/jdbc/typing_deduping/JdbcDestinationHandler.kt

```diff
@@ -340,22 +348,28 @@ abstract class JdbcDestinationHandler<DestinationState>(
         }
     }
 
-    private fun isAirbyteRawIdColumnMatch(existingTable: TableDefinition): Boolean {
-        return existingTable.columns.containsKey(JavaBaseConstants.COLUMN_NAME_AB_RAW_ID) &&
-            toJdbcTypeName(AirbyteProtocolType.STRING) ==
-                existingTable.columns[JavaBaseConstants.COLUMN_NAME_AB_RAW_ID]!!.type
+    protected open fun isAirbyteRawIdColumnMatch(existingTable: TableDefinition): Boolean {
+        return toJdbcTypeName(AirbyteProtocolType.STRING)
+            .equals(
```

Comment:
the kotlin `==` is the java `equals()` (https://kotlinlang.org/docs/operator-overloading.html#equality-and-inequality-operators). Let's keep the `==`

Reply:
is it possible to pass ignoreCase into the operator overload?

Reply:
oh wait, I hadn't realized that the `ignoreCase` was a parameter of equals. you can "ignore" my comment